WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
+  serialize :combined_round_cutoff, CombinedRoundCutoff
+  serialize :next_round_cutoff, Cutoff
```

Comment:
@laura-o, because the is very closely related to https://github.com/thewca/wca-regulations/issues/423, which might result in this not even being called a cutoff anymore. Laura, it would be really awesome if we can get our code and the regulations in sync with each other here. Any ideas?